### ISSUE TITLE
与原文公式(7)对比grad少乘了一个\alpha_t

### ISSUE THREAD

**Issue by zhuzhu18** [Zeqiang-Lai/OpenDMD#11]
另外损失函数应该写为：loss = 0.5 * torch.sum(torch.square(latents - stopgrad(latents - grad)))吧
